Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -211,7 +232,15 @@ void JitArm64::bcctrx(UGeckoInstruction inst)
 
   LDR(INDEX_UNSIGNED, WA, PPC_REG, PPCSTATE_OFF(spr[SPR_CTR]));
   AND(WA, WA, 30, 29);  // Wipe the bottom 2 bits.
-  WriteExit(WA);
+
+  if (inst.LK_3)
+  {
+    WriteLKExit(WA, js.compilerPC + 4);
```

Comment:
Unrelated to the main part of this PR, but constant propagation might have resolved the jump address to a constant.